lib/irc-appservice.js

```diff
@@ -36,6 +37,9 @@ module.exports.configure = function(config) {
         ident.configure(config.ident);
         ident.run();
     }
+    if (config.uniqueIPv6.prefix) {
+        ircClient.ip6Prefix = config.uniqueIPv6.prefix;
```

Comment:
Everything in the config will already be assigned to the `IrcServer` - see https://github.com/matrix-org/matrix-appservice-irc/blob/master/lib/irclib/server.js#L17 - Please get the IPv6 prefix from there via an accessor like https://github.com/matrix-org/matrix-appservice-irc/blob/master/lib/irclib/server.js#L61